docs/shortfin/llm/user/shortfin_with_sglang_frontend_language.md

```diff
@@ -79,57 +76,73 @@ import sglang as sgl
 
 from sglang.lang.chat_template import get_chat_template
 
-backend = sgl.Shortfin(chat_template=get_chat_template("llama-3-instruct"), base_url="http://localhost:8000", ) # Change base_url if running at different address
+backend = sgl.Shortfin(chat_template=get_chat_template("llama-3-instruct"), base_url="http://10.158.231.134:80", ) # Change base_url if running at different address
 
 sgl.set_default_backend(backend)
 
 @sgl.function
 def multi_turn_question(s, question_1, question_2):
      s += sgl.user(question_1)
-     s += sgl.assistant(sgl.gen("answer_1", max_tokens=256))
+     s += sgl.assistant(sgl.gen("answer_1", max_tokens=50))
```

Comment:
Should this be an argument passed in as well somewhere?

Reply:
Yeah, maybe something like:

### Run example

You can copy and paste the following examples into your interpreter. 

In these examples we will set the `max_tokens` to 50, allowing for a maximum of 50 tokens per completion request. For longer or shorter requests, `max_tokens` can be modified to a higher or lower value.

Reply:
There are other options too, right? I'd put a reference page up somewhere, like these: 

* https://docs.vllm.ai/en/latest/serving/openai_compatible_server.html#cli-reference
* https://docs.vllm.ai/en/latest/serving/deploying_with_helm.html#values

Reply:
Added description for `max_tokens` and `temperature` (params that can be passed to shortfin through SGLang).

Added `Server Options` section to `llama_serving.md`, detailing the available flags we have when running shortfin server